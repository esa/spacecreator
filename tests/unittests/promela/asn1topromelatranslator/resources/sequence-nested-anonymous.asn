Sequence-nested DEFINITIONS ::=
BEGIN

Outer-sequence ::= SEQUENCE {
   memberSequence SEQUENCE {
       memberInt INTEGER (0 .. 10)
   }
}

END
