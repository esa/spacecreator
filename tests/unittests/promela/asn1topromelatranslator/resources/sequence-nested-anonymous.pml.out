inline Outer_sequence_memberSequence_memberInt_generate_value(Outer_sequence_memberSequence_memberInt_gv)
{
    int Outer_sequence_memberSequence_memberInt_tmp;
    if
    ::  atomic {
        select (Outer_sequence_memberSequence_memberInt_tmp : 0 .. 10);
    }
    fi;
    Outer_sequence_memberSequence_memberInt_gv = Outer_sequence_memberSequence_memberInt_tmp;
}
inline Outer_sequence_memberSequence_generate_value(Outer_sequence_memberSequence_gv)
{
    Outer_sequence_memberSequence_memberInt_generate_value(Outer_sequence_memberSequence_gv.memberInt);
}
inline Outer_sequence_generate_value(Outer_sequence_gv)
{
    Outer_sequence_memberSequence_generate_value(Outer_sequence_gv.memberSequence);
}
